packages/sdk/src/modules/did/index.ts

```diff
@@ -92,15 +92,15 @@ export class Did extends Base {
       );
 
       const nonce = await this._getNonce(keyPair.address);
-      await this._newSignTx({nonce, address: keyPair, extrinsics: attributeExtrinsic});
+      const eventData = await this._newSignTx({nonce, address: keyPair, extrinsics: attributeExtrinsic});
```

Comment:
Returning back the peaqEvent created in the ./base/index.ts file. The event contains the hash of the block the transaction executed on. 

Any other places in the sdk we should return this block hash?